metals/src/main/scala/scala/meta/internal/metals/MetalsLspService.scala

```diff
@@ -1914,8 +1917,6 @@ class MetalsLspService(
         if (!buildTools.isAutoConnectable) {
           warnings.noBuildTool()
         }
-        // wait for a bsp file to show up
-        fileWatcher.start(Set(folder.resolve(".bsp")))
```

Comment:
This was causing memory leak in tests.

If fileWatcher was started multiple times without canceling between them, only the last thread would be canceled.
Now we watch .bsp directory by default (see `Configs.scala`)